include_core/AtomicSupport.hpp

```diff
@@ -440,19 +451,7 @@ class VM_AtomicSupport
 #endif /* defined(__riscv) */
 #elif defined(_MSC_VER) /* defined(__GNUC__) */
 		return (uint64_t)_InterlockedCompareExchange64((volatile __int64 *)address, (__int64)newValue, (__int64)oldValue);
-#elif defined(J9ZOS390) /* defined(_MSC_VER) */
-		 /* V1.R13 has a compiler bug and if you pass a constant as oldValue it will cause c-stack corruption */
-		 volatile uint64_t old = oldValue;
-#if defined(OMR_ENV_DATA64)
-		 /* Call __csg directly as csg() does not exist */
-		__csg((void*)&old, (void*)address, (void*)&newValue);
-		return old;
-#else /* defined(OMR_ENV_DATA64) */
-		/* __cds1 does not write the swap value correctly, cds does the correct thing */
-		cds((cds_t*)&old, (cds_t*)address, *(cds_t*)&newValue);
-		return old;
-#endif /* defined(OMR_ENV_DATA64) */
-#else /* defined(J9ZOS390) */
+#else /* defined(_MSC_VER) */
 #error "lockCompareExchangeU64(): unsupported platform!"
 #endif /* defined(__xlC__) */
```

Comment:
as per the [coding standard](https://github.com/eclipse-omr/omr/blob/3223d8d0d584575eafb5ec462250baae92cd6084/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif), `#endif` should have a comment describing the matching `#if`?
```suggestion
#endif /* defined(OMR_ARCH_POWER) && !defined(OMR_ENV_DATA64) */
```

Reply:
Thanks, I am seeing a bunch of other inconsistencies as well, so I will be correcting those too in the following commit.